rskj-core/src/test/java/co/rsk/core/BlockFactoryTest.java

```diff
@@ -291,44 +300,47 @@ void decodeBlockRskip110OffRskipUMMOnAndMergedMiningFieldsNullUmmRoot() {
         RLPList headerRLP = RLP.decodeList(encodedHeader);
         assertThat(headerRLP.size(), is(19));
 
-        Assertions.assertThrows(IllegalArgumentException.class, () -> factory.decodeHeader(encodedHeader, false));
+        assertThrows(IllegalArgumentException.class, () -> factory.decodeHeader(encodedHeader, false));
     }
 
     @Test
     void genesisHasVersion0() {
-        Assertions.assertEquals((byte) 0x0, factory.decodeBlock(genesisRaw()).getHeader().getVersion());
+        assertEquals((byte) 0x0, factory.decodeBlock(genesisRaw()).getHeader().getVersion());
     }
 
     @Test
-    void headerIsVersion0Before351Activation () {
+    void headerIsVersion0Before351Activation() {
         long number = 20L;
         enableRskip351At(number);
         BlockHeader header = factory.getBlockHeaderBuilder().setNumber(number - 1).build();
-        Assertions.assertEquals(0, header.getVersion());
+        assertEquals(0, header.getVersion());
     }
 
     @Test
-    void headerIsVersion1After351Activation () {
+    void headerIsVersion1After351Activation() {
```

Comment:
```suggestion
    void headerIsVersion1After351AndBefore535Activation() {
```
Just an idea, to make it explicit